pkg/serverpb/api.proto

```diff
@@ -26,6 +26,9 @@ service DKV {
   // CompareAndSet offers the standard CAS style transaction over a given
   // key. Intended to be used in concurrent workloads with less contention.
   rpc CompareAndSet (CompareAndSetRequest) returns (CompareAndSetResponse);
+
+  //GetKeySpaceSize returns the estimated number of keys the db
+  rpc GetKeySpaceSize (google.protobuf.Empty) returns (KeySpaceSizeResponse);
```

Comment:
Can be renamed to GetKeySpaceCount to avoid ambiguity between size and count, as most lsm based stores uses term size to denote mem/disk used in bytes

Reply:
IMO we should have consistency in name of the exposed command and the rpc method name.